### ISSUE TITLE
Pre-/post- deploy tasks

### ISSUE THREAD

**Issue by kfidelak94** [forcedotcom/devops-center-roadmap#38]
Users can specify tasks, scripts to run pre- or post- deployment.

**Comment by Damecek** [comment]
A common scenario is to run destructive changes pre- or post- deployment.

**Comment by SAxcell** [comment]
Yes!  We currently have custom fields in our Salesforce Enhancement records to track this info, but would love to put it where it belongs - on the work item

**Comment by jhownfs** [comment]
Yes! We need that. Need help to development that?
